modules/nomad-cloudinit-ubuntu-v1/templates/nomad_user_data.tpl

```diff
@@ -24,6 +24,12 @@ apt-get install -y "linux-image-$UNAME"
 apt-get update
 apt-get -y install docker-ce=17.03.2~ce-0~ubuntu-xenial
 
+# force docker to use userns-remap to mitigate CVE 2019-5736
+apt-get -y install jq
+tmp=$(mktemp)
+cp /etc/docker/daemon.json /etc/docker/daemon.json.orig
+jq '."userns-remap": "default"' /etc/docker/daemon.json > "$tmp" && mv "$tmp" /etc/docker/daemon.json
```

Comment:
i'm getting a syntax error testing this jq statement on my machine. The assignment operator in jq is `=`

Reply:
the syntax error has been fixed